docker/setup_configuration/data.yaml

```
@@ -0,0 +1,8 @@
+oidc_db_config_enable: true
+oidc_db_config_admin_auth:
+  oidc_rp_client_id: client-id
+  oidc_rp_client_secret: secret
+  endpoint_config:
+    oidc_op_authorization_endpoint: https://example.com/realms/test/protocol/openid-connect/auth
+    oidc_op_token_endpoint: https://example.com/realms/test/protocol/openid-connect/token
+    oidc_op_user_endpoint: https://example.com/realms/test/protocol/openid-connect/userinfo
```

The format for this changed in 0.21.1
```suggestion
  items:
  - identifier: admin-oidc
    oidc_rp_client_id: client-id
    oidc_rp_client_secret: secret
    endpoint_config:
      oidc_op_authorization_endpoint: https://example.com/realms/test/protocol/openid-connect/auth
      oidc_op_token_endpoint: https://example.com/realms/test/protocol/openid-connect/token
      oidc_op_user_endpoint: https://example.com/realms/test/protocol/openid-connect/userinfo
    userinfo_claims_source: userinfo_endpoint
```